README.md

```diff
@@ -34,6 +39,11 @@ To install ipywidgets from git, you will need [npm](https://www.npmjs.com/).
 
         pip install -v -e .
 
+2. Install the Jupyter Widgets nbextension
```

Comment:
Before installing the jupyter-js-widgets-nbextension, you have to go into jupyter-js-widgets and run `npm install` (and maybe `npm run build`).